apply/apply.go

```diff
@@ -79,7 +80,8 @@ func Apply(fs afero.Fs, conf *v2.Config, tmp *templates.T, upgrade bool) error {
 		return errs.WrapUser(e, "unable to apply envs")
 	}
 
-	e = applyGlobal(fs, p.Global, &tmp.Global, &tmp.Common)
+	x := tmp.Components[v2.ComponentKindTerraform]
+	e = applyGlobal(fs, p.Global, &x, &tmp.Common)
```

Comment:
```suggestion
	tfBox = tmp.Components[v2.ComponentKindTerraform]
	e = applyGlobal(fs, p.Global, &tfBox, &tmp.Common)
```
not sure if that makes sense